server/sonar-server/src/main/java/org/sonar/server/permission/PermissionPrivilegeChecker.java

```diff
@@ -37,13 +37,9 @@ public static void checkGlobalAdmin(UserSession userSession, String organization
       .checkOrganizationPermission(organizationUuid, SYSTEM_ADMIN);
   }
 
-  /**
-   * @deprecated does not support organizations. Replaced by {@link #checkProjectAdmin(UserSession, String, Optional)}
-   */
-  @Deprecated
-  public static void checkProjectAdminUserByComponentKey(UserSession userSession, @Nullable String componentKey) {
+  public static void checkProjectAdminUserByComponentUuid(UserSession userSession, @Nullable String componentUuid) {
     userSession.checkLoggedIn();
-    if (componentKey == null || !userSession.hasComponentPermission(UserRole.ADMIN, componentKey)) {
+    if (componentUuid == null || !userSession.hasComponentUuidPermission(UserRole.ADMIN, componentUuid)) {
       userSession.checkPermission(SYSTEM_ADMIN);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "checkPermission"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)